deploy/charts/harvester-crd/README.md

```diff
@@ -0,0 +1,58 @@
+# Harvester CRD Helm Chart
+
+## Chart Details
+
+This chart includes following CRDs:
+
+- Harvester (Automatically generated by [this script](https://github.com/harvester/harvester/blob/6557fdfd178b149b2af48ca0c8274bd3234c2066/scripts/generate-manifest))
+- KubeVirt
+- csi-snapshotter
+- whereabouts
+
+## Introduction
+
+Harvester depends on several components that provide the CRDs it needs. However, most of these CRDs must be created manually. To simplify management, Harvester uses an [independent folder to manage all CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-2-separate-charts).
+
+That said, not all CRDs can be managed by Helm. For example, the `cdi` CRD is managed by KubeVirt, but KubeVirt does not create the CRD automatically. Therefore, Harvester must create it manually. These CRDs are applied in the [`upgrade_manifests.sh`](https://github.com/harvester/harvester/blob/50da36ac3b751c1a1dbfc8d25e5499a4c6216450/package/upgrade/upgrade_manifests.sh#L1358) script.
+
+Additionally, Longhorn CRDs are not managed here either because Longhorn handles its own CRD creation process.
+
+> If you're interested in this change, please check [this issue](https://github.com/harvester/harvester/issues/8163) for more details.
+
+## For Developers
+
+### How to bump CRDs
+
+Bumping a CRD is quite straightforward. Developers just need to download the CRD from the correct source and place it in the `harvester-crds` folder.
+
+For example, to bump the KubeVirt CRD to version v1.5.1, download it from:
+
+`https://github.com/kubevirt/kubevirt/releases/download/v1.5.1/kubevirt-operator.yaml`
+
+Search for `CustomResourceDefinition` in the file, copy the relevant CRD, and save it with a versioned filename like `crd-kubevirt-1.5.1.yaml`. Be sure to delete the old version.
+
+If the CRD is similar to `cdi`, copy it to the `package/upgrade/extra_manifests` folder, and update the [`upgrade_manifests.sh`](https://github.com/harvester/harvester/blob/50da36ac3b751c1a1dbfc8d25e5499a4c6216450/package/upgrade/upgrade_manifests.sh#L1358) script like this:
+
+```sh
+for manifest in /usr/local/share/extra_manifests/{your_new_crds}/*.yaml; do
+    echo "Applying $manifest"
+    kubectl apply -f "$manifest"
+done
+```
+
+In this case, there's no need to put it under the harvester-crd folder.
+
+## License
+Copyright (c) 2025 [Rancher Labs, Inc.](http://rancher.com)
```

Comment:
Should we follow the license on README?

https://github.com/harvester/harvester/blob/8501f6789bde419dd6724976679ca5f5cf5f0ff8/README.md?plain=1#L136-L137